Packs/CrowdStrikeFalconSandbox/Integrations/CrowdstrikeFalconSandboxV2/CrowdstrikeFalconSandboxV2.py

```diff
@@ -0,0 +1,532 @@
+    response = client.search(query_args)
+
+    key_name_changes = {'job_id': 'JobID',
+                        'sha256': 'SHA256',
+                        'environment_id': 'environmentId',
+                        'threat_score': 'threatscore',
+                        'environment_description': 'environmentDescription',
+                        'submit_name': 'submitname',
+                        'analysis_start_time': 'start_time'}
+
+    # each indicator needs its own CR since indicators is deprecated.
+    def convert_to_file_res(res):
+        return CommandResults(readable_output=' ', indicator=BWCFile(res, key_name_changes, False, size=res['size'],
+                                                                     sha256=res['sha256'],
+                                                                     dbot_score=Common.DBotScore.NONE,
+                                                                     extension=res['type_short'],
+                                                                     name=res['submit_name'],
+                                                                     malware_family=res['vx_family']))
+
+    return [CommandResults(
+        raw_response=response,
+        outputs_prefix='CrowdStrike.Search',
+        outputs_key_field='sha256',
+        outputs=response['result'],
+        readable_output=tableToMarkdown("Search Results:", response['result'],
+                                        ['submit_name', 'verdict', 'vx_family', 'threat_score', 'sha256', 'size',
+                                         'environment_id', 'type', 'type_short', 'analysis_start_time'],
+                                        removeNull=True, headerTransform=underscore_to_space)
+    ), *[convert_to_file_res(res) for res in response['result']]]
+
+
+@poll('cs-falcon-sandbox-scan')
+def crowdstrike_scan_command(client: Client, args: Dict[str, Any]) -> PollResult:
+    hashes = args['file'].split(',')
+    scan_response = client.scan(hashes)
+
+    def file_with_bwc_fields(res):
+        return CommandResults(readable_output=' ', indicator=BWCFile(res, {
+            'sha1': 'SHA1',
+            'sha256': 'SHA256',
+            'md5': 'MD5',
+            'job_id': 'JobID',
+            'environment_id': 'environmentId',
+            'threat_score': 'threatscore',
+            'environment_description': 'environmentDescription',
+            'submit_name': 'submitname',
+            'url_analysis': 'isurlanalysis',
+            'interesting:': 'isinteresting',
+            'vx_family': 'family'}, False, size=res['size'], file_type=res['type'], sha1=res['sha1'],
+            sha256=res['sha256'], md5=res['md5'], sha512=res['sha512'], name=res['submit_name'],
+            ssdeep=res['ssdeep'], malware_family=res['vx_family'],
+            dbot_score=get_dbot_score(res['sha256'], res['threat_level'])))
+
+    command_result = [CommandResults(outputs_prefix='CrowdStrike.Report',
```

Comment:
hey, thats correct since indicators is false, and this command can have more than 1 file result. I can probably add some special logic in the case that we only have 1 file returned. Lets discuss?